src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Indentation/IndentationOptions.cs

```diff
@@ -14,6 +14,7 @@ namespace Microsoft.CodeAnalysis.Indentation
         SyntaxFormattingOptions FormattingOptions,
         AutoFormattingOptions AutoFormattingOptions)
     {
```

Comment:
Thinking out loud: Isn't this going to cause a conflict where the same internal type `Microsoft.CodeAnalysis.Indentation.IndentationOptions` is available from both the CodeStyle and Workspaces layer? If you get the build to succeed, then no issues.